storage_v2/proto/storage.proto

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) 2024 The Jaeger Authors.
```

Comment:
We can remove this from this PR and include it in gRPC Remote Storage implementation later.